debian/changelog

```diff
@@ -10,6 +10,13 @@ ubuntu-themes (16.11+ubports) xenial; urgency=medium
 
  -- UBports auto importer <infra@ubports.com>  Sun, 07 Jan 2018 23:13:42 +0100
 
+ubuntu-themes (16.10+17.04.20170126-0ubuntu1) zesty; urgency=medium
```

Comment:
The change here is OK but you need to add a new entry on top as well. It needs to increase the version number to +ubports1. This change should be done with the dch tool to keep the strict formatting. If you are unsure how to use it we can also do this.

Reply:
if you'd like to help me with that tool, I'd be glad.
otherwise you can do it by yourself.
as you prefer

Reply:
Ok, I can do it myself (I hope) but what should I write there?
`ubuntu-themes (18.10+ubports1) xenial; urgency=medium`? is `18.10` ok?
di I need to put the name of smb before the changelog? like `[ Iain Lane ]`...
in the changelog (after `*`) what should I write? `upstream 201703 suru icons`?
in the user info? is infra@ubports.com ok or should I put my info? I think not as I didn't do quite nothing (just the PR)...
thanks

Reply:
Smth like this:
```
ubuntu-themes (18.10+ubports1) xenial; urgency=medium

  * Update Suru styles from upstream ....

 -- Mike <email>  <Date_exact_format>

```
Its ok to put your name and email, I mean you did fix a few things also

Reply:
its important to increase the version to ubports1...

Reply:
> Its ok to put your name and email, I mean you did fix a few things also
well, no. I just took the latest files from LP and pushed here without the PNGs...

Reply:
Ok still put your identity. We need to bump the version and you did it so yeah ;)